backend/unitTests/chatAttackTest.py

```
@@ -0,0 +1,243 @@
+
+BASE_URL = "http://localhost:8000"
+
+def test_normal_chat_query():
+    """Test normal chat functionality - asking about user's inventory"""
+    chat_data = {
+        "username": "test_user",
+        "text": "How many items do I have in my inventory?"
+    }
+    
+    try:
+        response = requests.post(f"{BASE_URL}/api/chat", json=chat_data)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)